lms/views/decorators/h_api.py

```diff
@@ -106,6 +111,41 @@ def wrapper(context, request):
     return wrapper
 
 
+def upsert_group_info(wrapped):
```

Comment:
As per previous comment, would this be better as a private function in the `upsert_course_group` decorator? It would simplify the diff quite a bit too.